File: Service/GroupMembershipManagement/Hosts/WebApi/WebApi.Models/DTOs/Destination.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

namespace WebApi.Models.DTOs
{
    public class Destination
    {
        public Destination(Guid objectId, string name, string email)
        {
            Id = objectId;
            Name = name;
            Email = email;
        }

        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
    }
}

File: UI/web-app/src/models/Destination.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

import { GroupSettings } from './GroupSettings';

export interface Destination {
  id?: string;
  name?: string;
  channelId?: string;
  channelName?: string;
  type: string;
  endpoints?: string[] | undefined;
  email?: string;
  groupSettings?: GroupSettings | undefined;
};